### ISSUE TITLE
Hyperlinks in captions do not work in IE11 and Edge

### ISSUE THREAD

**Issue by touled** [fancyapps/fancybox#1696]
Hi!

Hyperlinks in image captions do not work in Internet Explorer 11 and Edge. On other common browsers (Chrome, Firefox, Opera) links work as expected.
You can see this effect in [CodePen](https://codepen.io/fancyapps/pen/jwNbaW?editors=1010)

**Comment by fancyapps** [comment]
Hi,

This is related to IE11 bug -
https://stackoverflow.com/questions/26718261/ie-11-pointer-events-override

Thankfully, the fix is simple:

```CSS
.fancybox-caption a, .fancybox-caption button, .fancybox-caption select {
    position: relative;
}
```

**Comment by touled** [comment]
Thanks! This fix really works for both IE11 and Edge.

**Comment by fancyapps** [comment]
This fix was included in the latest release (v3.2.0).
